zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -0,0 +1,245 @@
+around 40 outputs, instead of around 1,300,000.
+
+# Privacy and Security Implications
+
+As development funding is a public good on the Zcash network, there are not
+relevant privacy concerns related to this proposal; all disbursement of
+(but not necessarily subsequent distribution) of development funds is 
+transparent and auditable by any participant in the network.
+
+#### Security implications of the One-Time Lockbox Disbursement
+
+After the activation block of this ZIP has been mined, all development funds
+previously accrued to the in-protocol lockbox will be held instead by a 3-of-5
+multisig address. The key-holders for this address will have the capability to
+spend these funds. Compromise or loss of 3 of these 5 keys would result in
+total loss of funds; as such, in the event of the compromise or loss of a
+single key, the Key-Holders MUST establish a new multisig key set and address,
+and transfer remaining unspent funds held by the original address before
+additional loss or compromise occurs.
+
+Because this is a one-time disbursement, additional key rotation infrastructure
+is not required.
+
+#### Security implications for Option 1
+
+Funds will continue to securely accrue to the Deferred Development Lockbox
+until a disbursement mechanism for the lockbox is implemented in a future
+network upgrade. Such a disbursement mechanism should be designed to include an
+in-protocol option for key rotation, such that it is not necessary to perform a
+network upgrade to recover from key loss or compromise, or to change the size
+of the signing set or the number of signatures required to reach threshold.
+
+#### Security implications for Option 2a
+
+As of the activation height of this ZIP, development funds will begin accruing
+as additional outputs spendable by a 3-of-5 multisig address on a
+block-by-block basis. Key-Holders will need to perform regular multiparty
+signing ceremonies in order to shield the resulting coinbase outputs. Each such
+signing ceremony involves shared spending authority being used to sign
+thousands of inputs to large shielding transactions; for practical reasons,
+this is often handled using a scripted process that has spending authority over
+these funds. This process is an attractive target for compromise; for this
+reason it is RECOMMENDED that address rotation (in this case, by means of
+hard-coding a sequence of addresses, each of which receives a time-bounded 
+subset of the block reward fractional outputs) be implemented, as was done
+for the ECC funding stream described in ZIP 1014 [^zip-1014].
+
+In the case of key compromise or loss, it may be necessary to perform an
+emergency Network Upgrade to perform a manual key rotation to ensure that
+future development funds are not lost.
+
+#### Security implications for Option 2b
+
+Due to the aggregation of funds recommended by Option 2b, it is no longer
+necessary use scripts with elevated privileges to perform shielding and/or
```

Comment:
"necessary to use"

Reply:
Addressed in https://github.com/zcash/zips/pull/994